packages/core/src/transport/batch.ts

```diff
@@ -35,18 +35,18 @@ export class Batch {
   }
 
   flush() {
-    if (this.bufferMessageCount !== 0) {
+    if (this.bufferMessagesCount !== 0) {
       const messages = this.pushOnlyBuffer.concat(objectValues(this.upsertBuffer))
-      this.request.send(messages.join('\n'), this.bufferBytesSize)
+      this.request.send(messages.join('\n'), this.bufferBytesCount)
       this.pushOnlyBuffer = []
       this.upsertBuffer = {}
-      this.bufferBytesSize = 0
-      this.bufferMessageCount = 0
+      this.bufferBytesCount = 0
+      this.bufferMessagesCount = 0
     }
   }
 
-  sizeInBytes(candidate: string) {
-    // Accurate byte size computations can degrade performances when there is a lot of events to process
+  countBytes(candidate: string) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I found `countBytes` a bit confusing since we have some `*BytesCount` at several places.
Unless it is just me, maybe we could use something like `evaluateBytesCount` or `computeBytesCount`

Reply:
Lets do `computeBytesCount`